packages/core/src/hydration/annotate.ts

```diff
@@ -161,13 +166,15 @@ function annotateLContainerForHydration(lContainer: LContainer, context: Hydrati
  * @param appRef An instance of an ApplicationRef.
  * @param doc A reference to the current Document instance.
  */
-export function annotateForHydration(appRef: ApplicationRef, doc: Document) {
+export function annotateForHydration(appId: string, appRef: ApplicationRef, doc: Document) {
   const injector = appRef.injector;
   const isI18nHydrationEnabledVal = isI18nHydrationEnabled(injector);
 
   const serializedViewCollection = new SerializedViewCollection();
   const corruptedTextNodes = new Map<HTMLElement, TextNodeMarker>();
   const viewRefs = appRef._views;
+  const shouldReplay = injector.get(IS_EVENT_REPLAY_ENABLED, false);
```

Comment:
It'd be great to extract `false` into a const (add it to the `event_replay.ts` file), so that we can find it easier and also use in other places where it might be relevant.
```suggestion
  const shouldReplayEvents = injector.get(IS_EVENT_REPLAY_ENABLED, EVENT_REPLAY_ENABLED_DEFAULT);
```